tests/modular_pipelines/flux/test_modular_pipeline_flux.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+# Copyright 2025 HuggingFace Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import tempfile
+import unittest
+
+import numpy as np
+import PIL
+import torch
+
+from diffusers.image_processor import VaeImageProcessor
+from diffusers.modular_pipelines import (
+    FluxAutoBlocks,
+    FluxKontextAutoBlocks,
+    FluxKontextModularPipeline,
+    FluxModularPipeline,
+    ModularPipeline,
+)
+from diffusers.modular_pipelines.flux.modular_blocks import (
+    AUTO_BLOCKS_KONTEXT,
+    FluxKontextAutoVaeEncoderStep,
+    FluxKontextProcessImagesInputStep,
+    FluxKontextVaeEncoderStep,
+    FluxVaeEncoderDynamicStep,
+)
+from diffusers.modular_pipelines.modular_pipeline_utils import InsertableDict
+
+from ...testing_utils import floats_tensor, torch_device
+from ..test_modular_pipelines_common import ModularPipelineTesterMixin
+
+
+# Because we should disable `auto_resize` during tests.
+FluxKontextVaeEncoderBlocks = InsertableDict(
+    [
+        ("preprocess", FluxKontextProcessImagesInputStep(_auto_resize=False)),
```

Comment:
For facilitating `_auto_resize=False` in testing. 

We don't take the `_auto_resize` argument as a call argument from the user when using Kontext modular.